core/src/main/scala/anorm/Column.scala

```diff
@@ -713,10 +713,11 @@ sealed trait JavaTimeColumn {
   implicit val columnToInstant: Column[Instant] = nonNull { (value, meta) =>
     val MetaDataItem(qualified, _, _) = meta
     value match {
+      case ts: java.sql.Timestamp => Right(ts.toInstant)
```

Comment:
That's handled by `case TimestampWrapper1(ts)` so why this duplicate?

Reply:
**TimestampWrapper1** and **TimestampWrapper2** check the existence of respectively fields **getTimestamp: java.sql.Timestamp** or **timestampValue: java.sql.Timestamp**

but **java.sql.Timestamp** has no such fields

cf screen of my IDE (which might not be the best proof, I agree)


![image](https://user-images.githubusercontent.com/5735335/103787879-7bca9300-503e-11eb-930c-6ea30b2d5f6b.png)


![image](https://user-images.githubusercontent.com/5735335/103787923-88e78200-503e-11eb-99c2-a25f77bc3447.png)



I've just checked the behavior by printing the class name of the value in each branch, after commenting my added line:

```scala
value match {
      //      case ts: java.sql.Timestamp => Right(ts.toInstant)
      case time: Long => Right(Instant ofEpochMilli time)
      case TimestampWrapper1(ts) =>
        println("matching TimestampWrapper1 " + className(value))
        Right(ts.toInstant)
      case TimestampWrapper2(ts) =>
        println("matching TimestampWrapper2 " + className(value))
        Right(ts.toInstant)
      case date: java.util.Date =>
        println("matching date " + className(value))
        Right(Instant ofEpochMilli date.getTime)
      case _ => Left(TypeDoesNotMatch(s"Cannot convert $value: ${className(value)} to Java8 Instant for column $qualified"))
    }
```

and I can see

> matching date java.sql.Timestamp

Therefore, as it, **TimestampWrapper**s cannot match **java.sql.Timestamp**, which might be confusing